serdaSimus/assignments/01TwoPointersPattern/lc1/lc1.js

```diff
@@ -0,0 +1,75 @@
+// Problem Statement #
+// Given an array of sorted numbers and a target sum, find a pair in the array whose sum is equal to the given target.
+
+// Write a function to return the indices of the two numbers (i.e. the pair) such that they add up to the given target.
+
+/*
```

Comment:
Don't forget to write an approach, with inputs, outputs, and detailed lines about what your code will do.